src/js/components/CreateServiceModalServicePickerOption.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import Panel from './Panel';
+
+function CreateServiceModalServicePickerOption({icon, title, onOptionSelect}) {
+  return (
+    <Panel className="panel-interactive clickable"
+      contentClass={[
+        'horizontal-center panel-cell flush-top',
+        {
+          'panel-cell-short': false,
+          'panel-cell-shorter': false
+        }
+      ]}
+      heading={(
+        <div className="icon icon-jumbo icon-image-container icon-app-container">
```

Comment:
Could we make this more composable with more components, so you can add the content you like?
```js
<CreateServiceModalServicePickerOption
  offset={true}
  onOptionSelect={this.handleServiceSelect.bind(null, {type: 'app'})} />
  <CreateServiceModalServicePickerOptionImage  src={defaultServiceImage} />
  <CreateServiceModalServicePickerOptionContent>
    Single Container
  </CreateServiceModalServicePickerOptionContent>
</CreateServiceModalServicePickerOption>
```

We could then look for `CreateServiceModalServicePickerOptionImage` and `CreateServiceModalServicePickerOptionContent` in this component and add them to the panel